components/storage/blockdevice/COMPONENT_SPIF/TESTS/block_device/spif/main.cpp

```diff
@@ -131,9 +131,9 @@ void test_spif_random_program_read_erase()
     delete[] read_block;
 }
 
-void test_spif_unaligned_program()
+void test_spif_unaligned_erase()
 {
-    utest_printf("\nTest Unaligned Program Starts..\n");
+    utest_printf("\nTest Unaligned Erase Starts..\n");
 
     SPIFBlockDevice blockD(MBED_CONF_SPIF_DRIVER_SPI_MOSI, MBED_CONF_SPIF_DRIVER_SPI_MISO, MBED_CONF_SPIF_DRIVER_SPI_CLK,
```

Comment:
Better not use a camel case name (blockD) here.

Reply:
will fix